src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,99 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private final String firstName = "apple";
+    private final int firstAmount = 5;
+    private final String secondName = "banana";
+    private final int secondAmount = 10;
+    private final String thirdName = "pineapple";
+    private final int thirdAmount = 15;
+    private final FruitDao fruitDao = new FruitDaoImpl();
+
+    @Test
+    public void addFruit_Ok() {
+        Storage.fruits.clear();
+        Fruit fruitThird = new Fruit();
+        fruitThird.setName(thirdName);
+        fruitThird.setAmount(thirdAmount);
+        fruitDao.addFruit(fruitThird);
+        Assert.assertTrue(Storage.fruits.contains(fruitThird));
+    }
+
+    @Test
+    public void getByName_Ok() {
+        Storage.fruits.clear();
+        Fruit fruitFirst = new Fruit();
+        fruitFirst.setName(firstName);
+        fruitFirst.setAmount(firstAmount);
+        fruitDao.addFruit(fruitFirst);
```

Comment:
you use the same block of code in several places but with different values, let's create a private method, that will be creating a fruit, passing needed values

Reply:
:)